dockerfiles/synapse/homeserver.yaml

```diff
@@ -93,6 +93,12 @@ rc_joins:
 
 federation_rr_transactions_per_room_per_second: 9999
 
+
+## Registration ##
+
+enable_registration: True
+registration_shared_secret: "$FZjMa&9fAAi9Xf[F)jAY[C#y?QwT[!qnBi+:ZLj.-)zVf]:C39H4Y99c$LPCh}{"
```

Comment:
Will remove :+1:

Not sure, I thought it might be so that I could register on the Synapse instance after hooking up an Element client to it, see the *Hook up Element (Matrix client) to the Complement server after a test runs* section in the issue description, but I've removed it and it still works.

Plus I see [`enable_registration` already in the config](https://github.com/matrix-org/complement/blob/64ce1536e5ceecbaefe06d8eea4c0911f8c74414/dockerfiles/synapse/homeserver.yaml#L8)